src/runtime_src/doc/toc/xbutil.rst

```diff
@@ -101,6 +101,10 @@ The command ``xbutil validate`` validates the installed card by running precompi
     - ``vcu``: Run decoder test (only applicable for specific platform).
     - ``quick``: Run first four tests (Aux connection, PCIE link, SC version and Verify kernel)
     - ``aie-pl``: Run AIE PL test
+    - ``aie-ps``: Run AIE PS test
+    - ``bw-ps``: Run bandwidth PS test
```

Comment:
This naming convention doesn't not following what was defined earlier (e.g., mem-bw and hostmem-bw).

In addition, since all of these names are customer visable, have they been approved by the PRT. 

If not, please do create a slide with these name changes ALONG with the existing validation tests name and present that at the next PRT meeting.